EventMixing/inc/Mu2eProductMixer.hh

```diff
@@ -140,14 +139,10 @@ namespace mu2e {
                           ExtMonFNALSimHitCollection& out,
                           art::PtrRemapper const& remap);
 
-    bool mixProtonBunchIntensity(std::vector<mu2e::ProtonBunchIntensity const*> const &in,
```

Comment:
Why was mixProtonBunchIntensity() removed when adding mixCosmicLivetime()?

Reply:
There is no use case for mixing this product.  Its only use case is in the main event stream of a mixing job.  I'm not sure of the history behind making it a potential mixin.